crates/e2e/src/setup/colocation.rs

```diff
@@ -38,7 +38,7 @@ pub fn start_driver(
 ) -> JoinHandle<()> {
     let config_file = config_tmp_file(format!(
         r#"
-# CI e2e tests run with hardhat, which doesn't support access lists.
+# CI e2e tests run with anvil, which does support access lists.
```

Comment:
So should we then set `disable-access-list-simulation` to false (or drop these two lines altogether)

Reply:
Dropped these lines after verifying that this would not break any tests. 👍